src/components/pages/Search.vue

```diff
@@ -15,102 +15,107 @@
             <button class="button" v-if="showNext" @click.prevent="showNextPage()">Next</button>
         </nav>
 
+      <!--
```

Comment:
@thadallender just pointing this out

Reply:
Ah, yes. This was part of an early attempt at adding action hooks that other plugins could hook into.